### ISSUE TITLE
Display command line output in a message box.

### ISSUE THREAD

**Issue by GitMensch** [OpenCobolIDE/OpenCobolIDE#322]
Checked with 4.7.3.dev24 (from freezed windows installation):
- dropping a file on OpenCobolIde.exe (or directly calling `OpenCobolIde.exe S:\mysrc\TEST.COB`) doesn't do anything (no output on command line, OCIDE is not started)
- calling `OpenCobolIde.exe /?` starts OCIDE with the error message "Failed to open path: " (without any path)
- calling `OpenCobolIde.exe -?` or `OpenCobolIde.exe -v` doesn't do anything (no output on command line, OCIDE is not started)

The only check I've done (before removing it): OCIDE 4.6.6 started and opened the file if dropping a file on OpenCobolIde.exe (or directly calling `OpenCobolIde.exe S:\mysrc\TEST.COB`)

**Comment by ColinDuquesnoy** [comment]
> dropping a file on OpenCobolIde.exe (or directly calling OpenCobolIde.exe S:\mysrc\TEST.COB) doesn't do anything (no output on command line, OCIDE is not started)

I'll recheck that, looks lilke a bug/regression.

> calling OpenCobolIde.exe /? starts OCIDE with the error message "Failed to open path: " (without any path)

This is a windows command line switch, not supported by OCIDE. Use --help instead. But OCIDE being a GUI application (in frozen mode) it will never output anything to the command line. I need to provide a CLI application which is planned for HE.

> calling OpenCobolIde.exe -? or OpenCobolIde.exe -v doesn't do anything

Those are not valid command line option for OCIDE. Here are the valid options:

usage: OpenCobolIDE.pyw [-h] [--verbose] [--runtime-env] [--cobc-runtime-env]
                        [--compile] [--conf CONF]
                        [files [files ...]]

Simple and lightweight COBOL IDE.

positional arguments:
  files               List of files to open, if any

optional arguments:
  -h, --help          show this help message and exit
  --verbose           Verbose mode will enable debug and info messages to be
                      shown in the application log
  --runtime-env       Display the application runtime environment.
  --cobc-runtime-env  Display the compiler runtime environment.
  --compile           Compile the specified files and exits.
  --conf CONF         Specify a configuration file (previously exported from
                      within the IDE (File->Export preferences))

I'll check why ocide does not start though.

**Comment by GitMensch** [comment]
I suggest to display the command line specific output (not supported options, output from `-v` and others) in a simple popup if in "GUI mode".

**Comment by ColinDuquesnoy** [comment]
> I suggest to display the command line specific output (not supported options, output from -v and others) in a simple popup if in "GUI mode".

Good idea. That will be for 4.7.4 though (I plan to release ocide 4.7.3 today).

**Comment by ColinDuquesnoy** [comment]
> dropping a file on OpenCobolIde.exe (or directly calling OpenCobolIde.exe S:\mysrc\TEST.COB) doesn't do anything (no output on command line, OCIDE is not started)

Rechecked. I could not reproduce the same issue as you: OCIDE starts but it shows the home page instead of the editor page. That is fixed and I added some additional messages to debug this if it does not work for you.

Regarding invalid commands, OCIDE does start and print this to stdout:

```
usage: OpenCobolIDE.pyw [-h] [--verbose] [--runtime-env] [--cobc-runtime-env]
                        [--compile] [--conf CONF]
                        [files [files ...]]
OpenCobolIDE.pyw: error: unrecognized arguments: -?
```

I'll leave it that way for now, this will be fixed in OCIDE 4.7.4 with the solution you suggested.

I'll release OCIDE 4.7.3 this evening.
